README.md

```diff
@@ -124,6 +126,21 @@ The user has the option to enable/disable the slack integration. To use the slac
 #### Go or no-go signal
 When the cerberus is configured to run in the daemon mode, it will continuosly monitor the components specified, runs a simple http server at http://0.0.0.0:8080 and publishes the signal i.e True or False depending on the components status. The tools can consume the signal and act accordingly.
 
+#### Node Problem Detector
+[node-problem-detector](https://github.com/kubernetes/node-problem-detector) aims to make various node problems visible to the upstream layers in cluster management stack
+##### Installation
+1. Create `openshift-node-problem-detector` namespace [ns.yaml](https://github.com/openshift/node-problem-detector-operator/blob/master/deploy/ns.yaml) with        `oc create -f ns.yaml`
+2. Add cluster role with `oc adm policy add-cluster-role-to-user system:node-problem-detector -z default -n openshift-node-problem-detector`
+3. Add security context constraints with `oc adm policy add-scc-to-user privileged system:serviceaccount:openshift-node-problem-detector:default
+`
+4. Edit [node-problem-detector-ds.yaml](https://gist.github.com/mohit-sheth/7a39410732781417b760152ab87c0181) to fit your environment.
```

Comment:
Can we use the daemonset spec provided in the upstream repo instead of maintaining our own? Any additional changes we had to add to get it up and running?

Reply:
just using it's own namespace and adding a toleration to land the pods on master nodes. I can add the above two changes as a note and can link the upstream ds